lib/client-assets.php

```diff
@@ -765,9 +770,14 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'colors'     => $color_palette,
 	);
 
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
-		. 'window._wpGutenbergEditor = wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
-		. '} );'
+	wp_add_inline_script(
+		'wp-editor',
+		(
+			'jQuery.when( wp.api.init(), wp.api.init( { versionString: \'gutenberg/v1/\' } ) ).done( function() {'
+			. 'wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
```

Comment:
I guess we should drop this line? Probably a merge conflict?

Reply:
Oops! Fixed in 5367b6e970cbb7e93c97a28f61459f7ffdee2f25.